tests/api/testdata/db_filler.py

```diff
@@ -194,11 +197,29 @@ def _solve_field_relations(self, endpoint: Endpoints, data: dict, field: Field,
                 ][-1]
                 field.f_type.schema = build_schema_by_json(current_config_log[field.name])
 
+        elif endpoint == Endpoints.RbacPolicy:
+            if field.name == "object":
+                role_fk = _data[related_field_name]
+                role = get_object_data(adcm=self.adcm, endpoint=endpoint.RbacAnyRole, object_id=role_fk)
+                field.f_type.payload = [
+                    {
+                        "id": self._get_adcm_object_id_by_object_type(object_type),
+                        "type": object_type,
+                    }
+                    for object_type in role["parametrized_by_type"]
+                ]
+
         else:
             raise NotImplementedError(f"Relations logic needs to be implemented for {endpoint} for field {field.name}")
 
         return _data, field
 
+    def _get_adcm_object_id_by_object_type(
+        self, object_type: Literal["cluster", "service", "component", "provider", "host"]
+    ) -> int:
+        """Get random created object by given type"""
+        return random.choice(get_endpoint_data(adcm=self.adcm, endpoint=Endpoints[object_type.capitalize()]))["id"]
```

Comment:
why you do not use enum in the field `object_type`?

Reply:
not the case